lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -437,11 +437,17 @@ fb_actions.toggle_browser = function(prompt_bufnr, opts)
   current_picker:refresh(finder, { reset_prompt = opts.reset_prompt, multi = current_picker._multi })
 end
 
---- Toggles all selections akin to |actions.toggle_all| but ignores "../" and "./".
+local function _get_ignore_values(current_picker)
+  local finder = current_picker.finder
+  local path_parent = tostring(Path:new(finder.path):parent())
+  return { "../", "./", path_parent, finder.path }
+end
+
+--- Toggles all selections akin to |actions.toggle_all| but ignores parent & current directory
 ---@param prompt_bufnr number: The prompt bufnr
 fb_actions.toggle_all = function(prompt_bufnr)
   local current_picker = action_state.get_current_picker(prompt_bufnr)
-  local ignore_values = { "../", "./" }
+  local ignore_values = _get_ignore_values(current_picker)
   action_utils.map_entries(prompt_bufnr, function(entry, _, row)
     if not vim.tbl_contains(ignore_values, entry.value) then
```

Comment:
The changes can probably be reduced to

```lua
    local parent_dir = Path:new(finder.path):parent():absolute()
    if not vim.tbl_contains({finder.path, parent_dir}, entry.value) then
```

Of course, analogously down below.